assembly/src/main/dist/docker/build.sh

```diff
@@ -14,10 +14,10 @@ rm -rf ../../../../target/
 
 # go to root of project and do clean, format, install and assembly
 cd ../../../../..
-MVN_VERSION=$(mvn -q -Dexec.executable=echo -Dexec.args='${project.version}' --non-recursive exec:exec)
+MVN_VERSION=$(xmllint --xpath "//*[local-name()='project']/*[local-name()='version']/text()" pom.xml)
```

Comment:
Like I also mentioned on the mailinglist: I may end up pulling this change out into a separate PR because it is blocking me from using this script.